### ISSUE TITLE
Getting a reference error, not able to use

### ISSUE THREAD

**Issue by 2Nipun** [farahat80/react-open-weather#50]
Im pulling this in a NextJS SSR app. Getting this error. I maybe doing something wrong. 

![image](https://user-images.githubusercontent.com/3709212/111569544-1b538280-8760-11eb-911b-9f2ef87d9018.png)

**Comment by WorldOfMaze** [comment]
I am getting this same error in Next js on a simple page without SSR using the code from the example:

```
import React from "react";
import ReactWeather, { useOpenWeather } from "react-open-weather";

const Weather = (props) => {
  const { data, isLoading, errorMessage } = useOpenWeather({
    key: "3d5...d13",
    lat: "32.99785136472139",
    lon: "-116.9235340276175",
    lang: "en",
    unit: "imperial", // values are (metric, standard, imperial)
  });
  return (
    <ReactWeather
      isLoading={isLoading}
      errorMessage={errorMessage}
      data={data}
      lang="en"
      locationLabel="Ramona, CA USA"
      unitsLabels={{ temperature: "F", windSpeed: "MPH" }}
      showForecast
    />
  );
};

Weather.propTypes = {};

export default Weather;
```

Message in Chrome  89.0.4389.114

![2021-04-10 20_53_03-Window](https://user-images.githubusercontent.com/8455132/114291813-305dc180-9a3f-11eb-8d80-7d94fe2d18fe.png)

And console log:

```
ReferenceError: self is not defined
    at Object.<anonymous> (D:\Users\steve\Development\wom-app\node_modules\react-open-weather\lib\ReactWeather.js:2:267)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.react-open-weather (D:\Users\steve\Development\wom-app\.next\server\pages\weather.js:170:18)
    at __webpack_require__ (D:\Users\steve\Development\wom-app\.next\server\pages\weather.js:23:31)
    at Module../src/pages/weather.js (D:\Users\steve\Development\wom-app\.next\server\pages\weather.js:109:76)
    at __webpack_require__ (D:\Users\steve\Development\wom-app\.next\server\pages\weather.js:23:31)
    at D:\Users\steve\Development\wom-app\.next\server\pages\weather.js:91:18
    at Object.<anonymous> (D:\Users\steve\Development\wom-app\.next\server\pages\weather.js:94:10)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
```

Running react 17..0.1, next 10.0.9 and react-open-weather 1.1.3.

**Comment by farahat80** [comment]
@2Nipun @WorldOfMaze do you guys have any sample, or any steps that i can use to reproduce the error i am not familiar with NextJS

**Comment by fgagneten-cm** [comment]
Hey @farahat80. Here is the bug fix https://github.com/farahat80/react-open-weather/pull/56

**Comment by farahat80** [comment]
@fgagneten-cm thanks for the fix, its merged and deployed now

@2Nipun and @WorldOfMaze can you guys try now with version 1.1.4

**Comment by farahat80** [comment]
Fixed in [#56](https://github.com/farahat80/react-open-weather/pull/56)
